config/forms/2021_2022.json

```diff
@@ -0,0 +1,306 @@
+{
+  "form_type": "lettings",
+  "start_year": 2021,
+  "end_year": 2022,
```

Comment:
I like this approach, I can see why it would make the form easy to change since it has to be changed on a yearly basis 👍